examples/gno.land/r/demo/users/users.gno

```diff
@@ -235,6 +235,20 @@ func GetUserByAddressOrName(input users.AddressOrName) *users.User {
 	return GetUserByAddress(std.Address(input))
 }
 
+// Get a list of user names starting from the given prefix. Limit the
+// number of results to maxResults. (This can be used for a name search tool.)
+func ListUsersByPrefix(prefix string, maxResults int) []string {
+	result := []string{}
+	name2User.Iterate(prefix, "", func(key string, value interface{}) bool {
+		if len(result) >= maxResults {
+			return true
+		}
+		result = append(result, key)
+		return false
+	})
+	return result
```

Comment:
```suggestion
	return result[:maxResults]
```

Reply:
We decided that we don't need to handle the edge case where the caller asks for zero results.